Formula/ponyc.rb

```diff
@@ -1,10 +1,9 @@
 class Ponyc < Formula
   desc "Object-oriented, actor-model, capabilities-secure programming language"
   homepage "http://www.ponylang.org"
-  url "https://github.com/ponylang/ponyc/archive/0.2.1.tar.gz"
-  sha256 "cb8d6830565ab6b47ecef07dc1243029cef962df7ff926140022abb69d1e554e"
-  revision 1
-  head "https://github.com/ponylang/ponyc.git"
+  url "https://github.com/ponylang/ponyc.git",
+    :revision => "aafebac938273d4786c02dfb2ba9ec7a164675e7"
+  version "0.2.2-candidate"
```

Comment:
Can you comment on how you came up with this commit and the `0.2.2-candidate`? Just a word of caution, but if upstream really decides to do a `0.2.2` release, it will unfortunately compare less than `0.2.2-candidate`, so people won't get updated. We'd need to use `0.2.2-alpha` or something similar as a version to get the correct sorting order.